.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,32 @@
-## Summary / Title of the Pull Request
+<!-- 
 
-[summary]
+PLEASE ACKNOWLEDGE THE FOLLOWING BEFORE PROCEEDING:
 
-## Detailed Description of the Pull Request / Additional Comments
+If I delete this entire template and go my own path, the core team may close my issue without further explanation or engagement.
```

Comment:
I don't like the vibe of this. Some people will misbehave but every contributor will read this. Also I don't like just closing a PR. I am happy to answer the "malformed" ones.

Reply:
While I understand the sentiment, malformed PRs hurt us more than they do us good. Keeping in mind that we're gonna add documentation and releases. Having a template that everyone needs to conform to will help with 

- Creating release notes of new additions.
- Ease of review, we review the logic as the author intended to and do not prolong the PR with comments about, please conform or can you explain, or worse do full research on the technique covered.
- Ease of management and documentation.

Also, btw fun fact this phrase is taken from one of MS GitHub projects :D

Reply:
As discussed internally, I will remove this as suggested and keep it neutral. Thanks.